xonsh/tools.py

```diff
@@ -2018,11 +2018,14 @@ def format_color(string, **kwargs):
         return ansi_partial_color_format(string, style=style)
 
 
-def print_color(string, **kwargs):
+def print_color(string, color=None, **kwargs):
```

Comment:
What do you think about making it keyword-only? I'm unfamiliar with where `printx` is exposed or used, but given the name, I'd maybe have expected it to take similar arguments to `print` (i.e. `def print(*strs, **kwargs)`. If it's keyword-only, it'll be "one best way" to do it and won't be confused with `print()` semantics.

```suggestion
def print_color(string, *, color=None, **kwargs):
```